packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -144,15 +142,19 @@ async function returnPromise(
   })
 }
 
+// eslint-disable-next-line max-params -- helper function
 function getHTTPOptions(
   client: Client,
   postBody: Uint8Array,
   hostname?: string,
+  pathname?: string,
```

Comment:
These should be in an `options` so it's neater and easier to add more optional params in the future